action.yml

```diff
@@ -31,6 +31,9 @@ inputs:
   close-on-skipped:
     description: 'Close PR when no tests added. Use with comment-on-skipped to clarify this action'
     required: false 
+  github-pat:
```

Comment:
there must be an option to enable documentation & provide a page name for documentation in wiki